kernel/mutex.c

```diff
@@ -72,13 +73,17 @@ void _impl_k_mutex_init(struct k_mutex *mutex)
 	mutex->owner = NULL;
 	mutex->lock_count = 0;
 
+	sys_trace_void(SYS_TRACE_ID_MUTEX_INIT);
```

Comment:
Could we attach some information to the trace event in order to identify mutex in trace log?

Reply:
yes, those can be changed to a trace function that has the mutex information as a payload.